commit f09f857f6c3fe9d4f648fbcd22603b14612b58ab
Author: Gerald Turner <gturner@unzane.com>
Date:   Wed May 10 16:07:00 2017 -0700

    Install ‘/etc/strongswan.d/charon-systemd.conf’ with charon-systemd package.
    
    Upstream contains source ‘conf/options/charon-systemd.conf’ which is like
    ‘conf/options/charon-logging.conf’, however there is a bug with configure that
    it is not included in the install target.  Added quilt patch
    05_install-charon-systemd-conf.patch which fixes configure.

diff --git a/debian/charon-systemd.install b/debian/charon-systemd.install
index 3b62aade..6ab3af8f 100644
--- a/debian/charon-systemd.install
+++ b/debian/charon-systemd.install
@@ -1,2 +1,4 @@
+etc/strongswan.d/charon-systemd.conf
 lib/systemd/system/strongswan-swanctl.service
 usr/sbin/charon-systemd
+usr/share/strongswan/templates/config/strongswan.d/charon-systemd.conf
diff --git a/debian/patches/05_install-charon-systemd-conf.patch b/debian/patches/05_install-charon-systemd-conf.patch
new file mode 100644
index 00000000..67eb976f
--- /dev/null
+++ b/debian/patches/05_install-charon-systemd-conf.patch
@@ -0,0 +1,10 @@
+--- a/configure.ac
++++ b/configure.ac
+@@ -1724,6 +1724,7 @@ AM_COND_IF([USE_MEDSRV], [strongswan_options=${strongswan_options}" medsrv"])
+ AM_COND_IF([USE_SCEPCLIENT], [strongswan_options=${strongswan_options}" scepclient"])
+ AM_COND_IF([USE_PKI], [strongswan_options=${strongswan_options}" pki"])
+ AM_COND_IF([USE_SWANCTL], [strongswan_options=${strongswan_options}" swanctl"])
++AM_COND_IF([USE_SYSTEMD], [strongswan_options=${strongswan_options}" charon-systemd"])
+ 
+ AC_SUBST(strongswan_options)
+ 
diff --git a/debian/patches/series b/debian/patches/series
index 6d7cc1df..949de693 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -1,3 +1,4 @@
 01_fix-manpages.patch
 03_systemd-service.patch
 04_disable-libtls-tests.patch
+05_install-charon-systemd-conf.patch
